### PR TITLE
Fix/manipulator move group

### PR DESCRIPTION
**Pull request by gollth**
One potential fix to https://github.com/ros-planning/panda_moveit_config/issues/101. I think the main problem is that many configs were missing for the move group `manipulator`. Note, though, that with these fixes actually _two_ gizmos will be shown in RVIZ Motion planning when the `manipulator` is selected. Probably a bug in the Visualization
Screenshot from 2022-02-09 10-04-51

### CODE REVIEW

**Comment by gonzalocasas** [comment]
The change to `kinematics.yaml` makes a lot of sense to me. I just bumped into the fact that other planning groups besides the arm have no kinematics assigned.

About the other changes, I am not that familiar with those parts to say for sure.

File: launch/stomp_planning_pipeline.launch.xml [hunk 1]
```diff
@@ -21,5 +21,10 @@
   <!-- Add MoveGroup capabilities specific to this pipeline -->
   <!-- <param name="capabilities" value="" /> -->
 
-  <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml"/>
+  <rosparam command="load" file="$(find panda_moveit_config)/config/stomp_planning.yaml" ns="stomp/panda_arm"/>
+  <param name="stomp/panda_arm" value="panda_arm" />
```
Comment (rhaschke) [inline launch/stomp_planning_pipeline.launch.xml:25]: This needs to define the `group_name` parameter. Fixed upstream.

**Review by rhaschke — COMMENTED** [review]
Thanks for pointing out these issues. I noticed the missing interactive marker for the `manipulator` group as well and fixed it via c8950cc4cf68251e8959d6b6c7de221de2f4bdd9. I cherry-picked the two latter commits.

**Comment by rhaschke** [comment]
> Note, that with these fixes actually _two_ gizmos will be shown in RVIZ. Probably a bug in the visualization.

Actually, this was intended behavior: interactive markers are shown for all parent links that are part of the parent group.
However, I agree that this should be changed.

**Pull request closed without merging**